runtime/vm/JFRConstantPoolTypes.hpp

```diff
@@ -571,7 +567,7 @@ class VM_JFRConstantPoolTypes {
 
 	static UDATA stackTraceCallback(J9VMThread *vmThread, void *userData, UDATA bytecodeOffset, J9ROMClass *romClass, J9ROMMethod *romMethod, J9UTF8 *fileName, UDATA lineNumber, J9ClassLoader *classLoader, J9Class *ramClass, UDATA frameType)
 	{
-		VM_JFRConstantPoolTypes *cp = (VM_JFRConstantPoolTypes*) userData;
+		VM_JFRConstantPoolTypes *cp = (VM_JFRConstantPoolTypes *) userData;
```

Comment:
There should be no space after `)` (like lines 984, 987, 990, etc.).
Please add the missing spaces
* on line 601 (in `void **` and `void *`)
* on line 1014 (in `ClassEntry **`)
* on line 1042 (in `StackFrame *`)
* etc.